apps/passport-client/src/state.ts

```diff
@@ -51,11 +51,11 @@ export interface AppState {
 
   // Dynamic (in-memory-only) state-machine for sync of E2EE encrypted data.
   uploadedUploadId?: string;
-  uploadingUploadId?: string;
   downloadedPCDs?: boolean;
-  downloadingPCDs?: boolean;
   loadedIssuedPCDs?: boolean;
   loadingIssuedPCDs?: boolean;
+  completedFirstSync?: boolean;
+  shouldPollDownload?: boolean;
```

Comment:
on first read, the intention behind this variable is not clear to me

Reply:
I'll try extraDownloadRequested, though I think the purpose is only going to be fully clear with the documentation at the use site.